pylatex/base_classes/containers.py

```diff
@@ -71,7 +71,7 @@ def dumps_content(self, **kwargs):
     def _propagate_packages(self):
         """Make sure packages get propagated."""
 
-        for item in self.data:
+        for item in (self.data):
```

Comment:
That is where the preamble check for packages was added at first (until I realised my mistake and forgot to remove the brackets)